packages/odyssey-react-mui/src/labs/Switch.tsx

```diff
@@ -11,42 +11,164 @@
  */
 
 import {
+  ChangeEventHandler,
   memo,
-  SyntheticEvent,
   useCallback,
   useEffect,
   useMemo,
   useRef,
   useState,
 } from "react";
-import {
-  Switch as MuiSwitch,
-  SwitchProps as MuiSwitchProps,
-  FormControlLabel,
-} from "@mui/material";
+import { SwitchProps as MuiSwitchProps, FormLabel } from "@mui/material";
 import { useTranslation } from "react-i18next";
+import styled from "@emotion/styled";
 
-import { useOdysseyDesignTokens } from "../OdysseyDesignTokensContext";
 import { Box } from "../Box";
 import { FieldComponentProps } from "../FieldComponentProps";
 import { FieldHint } from "../FieldHint";
+import { CheckedFieldProps } from "../FormCheckedProps";
 import type { HtmlProps } from "../HtmlProps";
-import { useUniqueId } from "../useUniqueId";
+import { CheckIcon } from "../icons.generated";
 import { ComponentControlledState, getControlState } from "../inputUtils";
-import { CheckedFieldProps } from "../FormCheckedProps";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "../OdysseyDesignTokensContext";
+import { useUniqueId } from "../useUniqueId";
 
 const { CONTROLLED } = ComponentControlledState;
 
+const stripRemFromUnits = (remValue: string) => Number(remValue.split("r")[0]);
```

Comment:
Something like this should work and be more clear:
```suggestion
const stripRemFromUnits = (remValue: string) => Number(remValue.replace(/rem$/, ""))[0]);
```